desktop/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "Spectrum",
+  "description": "The community platform for the future.",
+  "version": "2.1.6",
+  "main": "src/main.js",
+  "private": true,
+  "dependencies": {
+    "electron-is-dev": "^0.3.0",
+    "electron-log": "^2.2.14",
+    "electron-updater": "^2.21.4"
+  },
+  "devDependencies": {
+    "concurrently": "^3.5.1",
+    "electron": "^1.8.4",
+    "electron-builder": "^20.8.1",
+    "wait-on": "^2.1.0"
+  },
+  "scripts": {
+    "dev": "cd ../ && concurrently \"cross-env BROWSER=none yarn dev:web\" \"wait-on http://localhost:3000 && electron ./desktop/src/main.js\"",
+    "package": "build --dir",
+    "package:mac": "build --mac",
+    "package:linux": "build --linux",
+    "package:win": "build --win --x64",
+    "package:all": "build -mwl",
+    "ship": "build -p always"
+  },
+  "build": {
+    "productName": "Spectrum",
+    "appId": "com.spectrum",
+    "files": [
+      "src/**/*",
+      "node_modules/**/*",
+      "resources/**/*",
+      "package.json"
+    ],
+    "directories": {
+      "buildResources": "resources",
+      "output": "release"
+    },
+    "mac": {
+      "category": "public.app-category.productivity",
+      "target": "default",
+      "icon": "resources/icons/mac/icon.icns"
+    },
+    "dmg": {
+      "background": "resources/background.tiff",
```

Comment:
I think I'd prefer a white background here with our purple logomark at the top, instead of the purple gradient with the white logomark at the top? The purple background really fights with the purple icon. /cc @uberbryn we'll probably need some assets for this.

<img width="652" alt="screen shot 2018-04-19 at 16 56 47" src="https://user-images.githubusercontent.com/7525670/38999695-acf79f8c-43f2-11e8-9cf6-9e8c2e67f58e.png">

Reply:
Totally agree this was just a test, need to work with @uberbryn for that !